packages/shared/providers.yaml

```diff
@@ -406,6 +406,16 @@ calendly:
     token_response_metadata:
         - owner
     docs: https://docs.nango.dev/integrations/all/calendly
+canny:
+    categories:
+        - feedback
+        - support
+    auth_mode: API_KEY
+    proxy:
+        base_url: https://canny.io/api/v1
+        data:
```

Comment:
You can adjust these lines to:
```
proxy:
        base_url: https://canny.io/api/v1
        query:
            apiKey: ${apiKey}
        verification:
            method: POST
            endpoint: /boards/list
  ```
  
  With `API_KEY` and `BASIC` auth_modes we encourage testing the credentials with the `proxy.verification` field. You can try this with an incorrect `api_key` to see if you will receive an error message after that add the correct key, when adding a new connection.